packages/authorization/README.md

```diff
@@ -42,6 +42,54 @@ Authorization decides if a **subject** can perform specific **action** on an
 
 - Run through voters/enforcers to make decisions
 
+## Decision matrix
+
+The final decision is controlled by voting results from authorizers and options
+for the authorization component.
+
+The following table illustrates the decision matrix with 3 voters and
+corresponding options.
+
+| Vote #1 | Vote # 2 | Vote #3 | Options                  | Final Decision |
+| ------- | -------- | ------- | ------------------------ | -------------- |
+| Deny    | Deny     | Deny    | **any**                  | Deny           |
+| Allow   | Allow    | Allow   | **any**                  | Allow          |
+| Abstain | Allow    | Abstain | **any**                  | Allow          |
+| Abstain | Deny     | Abstain | **any**                  | Deny           |
+| Deny    | Allow    | Abstain | {precedence: Deny}       | Deny           |
+| Deny    | Allow    | Abstain | {precedence: Allow}      | Allow          |
+| Allow   | Abstain  | Deny    | {precedence: Deny}       | Deny           |
+| Allow   | Abstain  | Deny    | {precedence: Allow}      | Allow          |
+| Abstain | Abstain  | Abstain | {defaultDecision: Deny}  | Deny           |
+| Abstain | Abstain  | Abstain | {defaultDecision: Allow} | Allow          |
+
+The `options` is described as follows:
+
+```ts
+export interface AuthorizationOptions {
+  /**
+   * Default decision if all authorizers vote for ABSTAIN
+   */
+  defaultDecision?: AuthorizationDecision.DENY | AuthorizationDecision.ALLOW;
+  /**
+   * Controls if Allow/Deny vote takes precedence and override other votes
```

Comment:
Maybe add one more comment:

E.g. when set `{precedence: AuthorizationDecision.DENY}`, if one voter votes for `DENY`, the final decision will be `DENY` and other voters won't execute.
Same when set it as `AuthorizationDecision.ALLOW`